public/pages/Snapshots/components/AddPrefixInput/AddPrefixInput.test.tsx

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from "react";
+import "@testing-library/jest-dom/extend-expect";
+import { render, screen, cleanup } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import AddPrefixInput from "./AddPrefixInput";
+
+
+const testProps = { getPrefix: jest.fn() };
+
+beforeEach(() => {
+  render(<AddPrefixInput {...testProps} />);
+});
+
+afterEach(() => {
+  cleanup();
+});
+
+describe("AddPrefixInput component", () => {
+  it("renders without error", () => {
+    expect(screen.getByText("Specify prefix for restored index names")).toBeInTheDocument();
+    expect(screen.getByTestId("prefixInput")).toBeInTheDocument();
+
+    cleanup();
```

Comment:
Curious what is this cleanup() doing here

Reply:
Turns out nothing in this case.  Will take it out in next PR.  Here's a link [https://testing-library.com/docs/react-testing-library/api/#cleanup](url) I think I used it because of the sentence under the example.